main.cpp

```diff
@@ -25,37 +25,58 @@ int main()
     while (!exit) {
         print_Welcome();
         std::cin >> num;
+        std::system("clear");
+
         switch (num) {
```

Comment:
It would be better to put whole switch in try block and after switch catch exceptions

Reply:
ok!